examples/gno.land/r/users/users.gno

```diff
@@ -247,14 +246,6 @@ func GetUserByName(name string) *User {
 	return userI.(*User)
 }
 
-// func GetAddrByName(name string) std.Address{
```

Comment:
you should not remove this code, it is used by external realms

Reply:
I wrote this function for test actually, So I delete it, thans moul